internal/cmd/common.go

```diff
@@ -167,17 +173,31 @@ func MonitorLoop(loopLogic MonitorLogic) {
 	ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGTERM)
 	defer stop()
 
+	if !loopLogic.Once() {
+		if err := server.StartMetricsServer(ctx, loopLogic.MonitorPort()); err != nil {
+			log.Fatalf("Failed to start Prometheus metrics server: %v", err)
+		}
+	}
+
 	config := loopLogic.Config()
 
 	// To get an immediate first tick, for-select is at the end of the loop
 	for {
 		fmt.Fprint(os.Stderr, "New monitor run at ", time.Now().Format(time.RFC3339), "\n")
+		server.IncLogIndexVerificationTotal()
 		inputEndIndex := config.EndIndex
 
 		prevCheckpoint, curCheckpoint, err := loopLogic.RunConsistencyCheck(ctx)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "error running consistency check: %v", err)
-			return
+			fmt.Fprintf(os.Stderr, "error running consistency check: %v\n", err)
+			if loopLogic.Once() {
+				return
+			} else {
+				if !strings.Contains(err.Error(), "consistency proofs can not be computed starting from an empty log") {
+					server.IncLogIndexVerificationFailure()
+				}
+				goto waitForTick
+			}
```

Comment:
I think this is similar to https://github.com/sigstore/rekor-monitor/pull/665#discussion_r2319463775 . We should probably handle this properly elsewhere? Is this just a way to workaround https://github.com/sigstore/rekor-monitor/pull/702 ? If that's the case, I think this shouldn't be needed if we merge that PR. Right?